frame/utility/src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub trait Trait: frame_system::Trait {
 	type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
 
 	/// The overarching call type.
-	type Call: Parameter + Dispatchable<Origin=Self::Origin> + GetDispatchInfo;
+	type Call: Parameter + Dispatchable<Origin=Self::Origin, PostInfo = PostDispatchInfo> + GetDispatchInfo;
```

Comment:
```suggestion
	type Call: Parameter + Dispatchable<Origin=Self::Origin, PostInfo=PostDispatchInfo> + GetDispatchInfo;
```

(apologies for the numerous style nits, but inconsistency _within a single line_ looks very bad to the eye.)